canonical-web-and-design/cn.ubuntu.com
# Multistage builds for cn.ubuntu.com

## Done

- [x] Based on ubuntu.com and jp.ubuntu.com
- [x] Copies over global-nav.js

## QA

- Build the image: `docker build --tag myimage --build-arg BUILD_ID=myfakeid .`
- Once image is built, `docker run -ti -p 8111:80 myimage`
- Check site is running smoothly at http://localhost:8111
- Check new id `myfakeid` is in the headers `curl -I http://localhost:8111`

## Issue

Fixes #2309
